advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/cyberglove_calibrator.py

```diff
@@ -90,36 +94,35 @@ def set_description(self, description):
         self.repaint()
 
 
-class StepSelector(QtGui.QWidget):
+class StepSelector(QtWidgets.QWidget):
 
     """
     The user can select the step to calibrate through this widget.
     """
 
     def __init__(self, parent, calibrer):
-        QtGui.QWidget.__init__(self, parent=parent)
+        QtWidgets.QWidget.__init__(self, parent=parent)
 
         self.current_step_name = None
         self.current_row = 0
         self.steps = {}
         self.steps_description = {}
 
-        self.frame = QtGui.QFrame()
+        self.frame = QtWidgets.QFrame()
         self.calibrer = calibrer
-        self.layout = QtGui.QVBoxLayout()
+        self.layout = QtWidgets.QVBoxLayout()
         self.layout.setSpacing(5)
 
-        self.title = QtGui.QLabel()
+        self.title = QtWidgets.QLabel()
         self.title.setText("Calibration Steps - 2 substeps by steps")
 
         self.step_describer = StepDescriber(self)
 
-        self.list = QtGui.QListWidget()
+        self.list = QtWidgets.QListWidget()
         first_item = self.refresh_list()
-        self.connect(self.list, QtCore.SIGNAL(
-            'itemClicked(QListWidgetItem*)'), self.step_choosed)
-        self.list.setViewMode(QtGui.QListView.ListMode)
-        self.list.setResizeMode(QtGui.QListView.Adjust)
+        self.list.itemClicked['QListWidgetItem*'].connect(self.step_choosed)
```

Comment:
You get this message when you start it on rqt or when you press a specific button?
I remember I could start it on rqt and press all buttons but it would do nothing because it said there is no glove